### DISCUSSION TITLE
ZuluSCSI for Roland SP-808 Sampler?

### DISCUSSION THREAD

**Discussion by gum-control** [ZuluSCSI/ZuluSCSI-firmware#435] [Q&A] [answered]
Hi,

There's a rumor that ZuluSCSI is making a version to support the Roland SP-808 sampler. Is this true?

The SP-808 Zip drive has been a notoriously difficult nut to crack. If the rumor is true it would be a pretty big deal to the many diehard SP-808 users out there. 

Thanks for any info...

**Comment by aperezbios** [comment, marked as answer]
@gum-control it isn't a ZuluSCSI, because the SP-808's built-in Zip drive isn't SCSI, but is actually ATAPI (IDE), but yes, I can confirm that we have ZuluIDE working with the SP-808. Please visit https://zuluide.com, and you can ask ZuluIDE-specific questions at https://github.com/ZuluIDE/ZuluIDE-firmware/discussions
